packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -216,19 +216,40 @@ export class SchemaFactory<
 	private readonly structuralTypes: Map<string, TreeNodeSchema> = new Map();
 
 	/**
-	 * Construct a SchemaFactory with a given scope.
+	 * Construct a SchemaFactory with a given {@link SchemaFactory.scope|scope}.
 	 * @remarks
-	 * There are no restrictions on mixing schema from different schema factories:
-	 * this is encouraged when a single schema references schema from different libraries.
-	 * If each library exporting schema picks its own globally unique scope for its SchemaFactory,
-	 * then all schema an application might depend on, directly or transitively,
-	 * will end up with a unique fully qualified name which is required to refer to it in persisted data and errors.
-	 *
-	 * @param scope - Prefix appended to the identifiers of all {@link TreeNodeSchema} produced by this builder.
-	 * Use of [Reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) or a UUIDv4 is recommended to avoid collisions.
-	 * You may opt out of using a scope by passing `undefined`, but note that this increases the risk of collisions.
+	 * There are no restrictions on mixing schema from different schema factories.
+	 * Typically each library will create one or more SchemaFactories and use them to define its schema.
 	 */
-	public constructor(public readonly scope: TScope) {}
+	public constructor(
+		/**
+		 * Prefix appended to the identifiers of all {@link TreeNodeSchema} produced by this builder.
+		 *
+		 * @remarks
+		 * Generally each developed independently library (possibly a package, but could also be part of a package or multiple packages developed together) should get its own globally unique `scope`.
```

Comment:
I'd probably change order slightly.

I'd start with "These are joined ..." sentence.
Then say - resulting schema identifier uniquely defines type in Tree instance ("globally" is not right, as not clear what is the scope of "globe"). And if two different types (having different layout) get same schema identifier, then really bad things happen (might be worth pointing to what exactly happens - might be useful for OCEs).

Only then I'd move to guidance on how to choose SchemaFactory.scope.

Reply:
Micrisoft (and some other users of GUIDs) likes to use "Global" to describe the scope in which identifiers which should never collide with anything else, for example https://learn.microsoft.com/en-us/previous-versions/windows/desktop/automat/globally-unique-identifier-guid-

I have rephrased the docs to avoid referring to global as it might not be obvious to people no subjected to using GUIDs all the time. I'm glad to know everyone is familiar with the term.

Also the identifier isn't for identifying types in a tree instance: we use the schema object identity for that. Instead its for associating schema with persisted data. The linked documentation on TreeNodeSchemaCore,identifier covers this, so I don't think adding details about it inline here is necessary.

Reply:
Microsoft is just being honest - we promise to empower everyone on the _planet_ to achieve more, but not everyone in the _universe_.